fastlane/lanes/release.rb

```diff
@@ -76,8 +76,8 @@
 
     push_to_git_remote(tags: false)
 
-    setbranchprotection(repository: GHHELPER_REPO, branch: "release/#{new_version}")
-    setfrozentag(repository: GHHELPER_REPO, milestone: new_version)
+    set_branch_protection(repository: GHHELPER_REPO, branch: "release/#{new_version}")
```

Comment:
Shouldn't we start using `copy_branch_protection(from: DEFAULT_BRANCH, to: "release/…")` instead?

Reply:
At first I thought it would be ok to keep as set, but I realize now apps seem to use `copy_branch_protection`; also, it will just follow whatever we have in `trunk`, which might be different per repo so it seems more consistent. Changed on a378a5ec01ffe91878bc75402005b0a1a1c10e4b.